client/src/modules/plan/planner/make-idea/make-idea.html

```diff
@@ -0,0 +1,31 @@
+<div class="make-idea-container flex-col flex-start flex-align-stretch">
+  <div class="make-idea-header"><span>Awesome! <br /> Let's get started</span></div>
+  <div class="make-idea-form-container">
+    <form ng-submit="$ctrl.submit()">
+      <div class="make-idea-input-block">
+        <label for="idea-title">Give your idea a name:</label>
+        <input class="awgl-form-input form-control" id="idea-title" ng-model="$ctrl.title"
```

Comment:
switch from using id's to using names and give the overall form a name, you can reference things within a form more easily when validation gets used with formName.fieldName